src/Storage/Commands.Storage.Test/Common/Cmdlet/NewAzureStorageContextTest.cs

```diff
@@ -20,9 +20,74 @@
 using Microsoft.WindowsAzure.Commands.Storage.Common.Cmdlet;
 using Microsoft.WindowsAzure.Storage;
 using Microsoft.WindowsAzure.Storage.Auth;
+using Microsoft.Azure.Commands.Common.Authentication;
+using Microsoft.Azure.Commands.Common.Authentication.Abstractions;
+using System.Collections.Generic;
 
 namespace Microsoft.WindowsAzure.Commands.Storage.Test.Common.Cmdlet
 {
+    class TestProfileProvider : AzureRmProfileProvider
```

Comment:
The cases are added to the new test you added.
The PR is updated, please check.